README.md

```diff
@@ -1366,7 +1366,7 @@ creating the job (number of replicas).
 Transformer Engine (TE) is a library for accelerating Transformer-based models on **NVIDIA Hopper GPUs**. It enables using 8-bit floating point (FP8) precision to provide better performance with lower memory utilization in both training and inference. NVIDIA open-sourced TE on [github](https://github.com/NVIDIA/TransformerEngine).
 
 In NeMo Framework, you can now use `fp8` to pre-train GPT models. For example, if you want to turn on `fp8` to pre-train a 
-GPT3 5B model, you can modify `gpt3/5b` training config inside `conf/training/gpt3/5b.yaml` file as following.
+GPT3 5B model, you can modify `gpt3/5b` training config inside `conf/training/gpt3/5b.yaml` file as following. We have already set the knobs associated with fp8 training accordingly, so all you need to change from the baseline yaml script for pre-training is setting `fp8` to `True`. In case of bf16 training, these knobs are ignored.
```

Comment:
Let's mention `transformer_engine=True` and `use_fp8=True` is what's needed

Reply:
Done